translations_to_weblate.sh

```diff
@@ -7,17 +8,8 @@ fi
 if [[ ! -z $2 ]] ; then
     output=$2
 fi
-
-echo "CHANGING FORMAT TO WEBLATE"
-cat "${target}" > weblate_translations.tmp
-awk '
-        BEGIN { print "{"; }
-        {   if (NR==TNR)
-                {print "\t\""$0"\": \""$0"\"";}
-              else
-                  {print "\t\""$0"\": \""$0"\",";}
-        }
-        END { print "}" }
-    ' weblate_translations.tmp >>"${output}"
+echo "-== CHANGING FORMAT TO WEBLATE (json) ==-"
+cat ${target} > weblate_translations.tmp
```

Comment:
Are these and variable-named elsewhere build products GitIgnored? Is there any locking against simultaneous runs polluting each other? (if not, we can try to enforce it in CI side then)

Reply:
no, there is no locking mechanism. variable-named could possibly be randomized at inputs, while others are relying to have only one running instance in a separate working directory. Do you think it would be needed? I can always use mktemp for that, but I liked static outputs more for debugging.
Good idea though to have it gitignored.